src/oph/ehoks/common/api.clj

```diff
@@ -77,18 +101,18 @@
    ::onr/oppija-not-found                bad-request-handler
    ::oi/opiskeluoikeus-not-found         bad-request-handler
    :shared-link-validation-error         bad-request-handler
-   :shared-link-expired                  (c-ex/with-logging
+   :shared-link-expired                  (with-logging
                                            (custom-ex-handler response/gone
                                                               :message)
                                            :warn)
-   :shared-link-inactive                 (c-ex/with-logging
+   :shared-link-inactive                 (with-logging
                                            (custom-ex-handler response/locked
                                                               :message)
                                            :warn)
    :not-found                            not-found-handler
    :unauthorized                         unauthorized-handler
 
-   ::c-ex/default                        c-ex/safe-handler})
+   ::c-ex/default                        safe-handler})
```

Comment:
Ehdotan, että `::c-ex/default`:n yliajamisen sijaan määritetään erillinen handler sille ExceptionInfo-tyypille, jonka  koski-client heittää, esim. `::koski/opiskeluoikeus-fetching-error`.

Reply:
vakavuus: toive

Reply:
Seuraava aikomus olikin itseasiassa toteuttaa näin :+1: